lib/webauthn/attestation_statement/android_safetynet.rb

```diff
@@ -40,6 +40,16 @@ def cts_profile_match?
         attestation_response.cts_profile_match?
       end
 
+      def attestation_trust_path
+        # FIXME: Change this to be the chain.
```

Comment:
Wondering when we want to do this - wait for level 2? Do we consider this breaking our API?

Reply:
Yeah, tricky one...

At first I thought we would wait for Level 2 stable Recommendation. But afterwards I became more inclined on fixing right away, mostly because it didn't seem like something that "changed" in the spec, but something it was "wrong" and inconsistent from the start which was "fixed".

Not strong on this matter though.
Do you think we should do otherwise?

Reply:
I think it'd be okay to return the full chain now, it makes more sense from an app developer perspective to have them all.